cmd/http-server/startcmd/start.go

```diff
@@ -430,6 +463,56 @@ func getOIDCParams(cmd *cobra.Command) (*oidcParameters, error) {
 	return params, nil
 }
 
+func getKeyParams(cmd *cobra.Command) (*keyParameters, error) {
+	params := &keyParameters{}
+
+	sessionCookieAuthKeyPath, err := cmdutils.GetUserSetVarFromString(cmd,
+		sessionCookieAuthKeyFlagName, sessionCookieAuthKeyEnvKey, false)
+	if err != nil {
+		return nil, fmt.Errorf("failed to configure session cookie auth key: %w", err)
+	}
+
+	params.sessionCookieAuthKey, err = parseKey(sessionCookieAuthKeyPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to configure session cookie auth key: %w", err)
+	}
+
+	sessionCookieEncKeyPath, err := cmdutils.GetUserSetVarFromString(cmd,
+		sessionCookieEncKeyFlagName, sessionCookieEncKeyEnvKey, false)
+	if err != nil {
+		return nil, fmt.Errorf("failed to configure session cookie enc key: %w", err)
+	}
+
+	params.sessionCookieEncKey, err = parseKey(sessionCookieEncKeyPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to configure session cooie enc key: %w", err)
+	}
+
+	return params, nil
+}
+
+func parseKey(file string) ([32]byte, error) {
+	const (
+		keyLen = 32
```

Comment:
I assume keyLen = 32 is because keys are used for block size 32 (AES256GCM). We should probably reference the block size of AES256GCM instead of hard coding it here.

Reply:
@Baha-sk where is the constant for AES256GCM block size defined?

Reply:
I thought I've seen AES having constant key sizes somewhere, but apparently not. So it's fine to keep it this way